v1.1/parallel-statement-execution.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Parallel Statement Execution
+summary: The parallel statement execution feature allows parallel execution of multiple independent SQL statements within a transaction.
+toc: false
+---
+
+CockroachDB supports parallel execution of independent [`INSERT`](insert.html), [`UPDATE`](update.html), [`UPSERT`](upsert.html), and [`DELETE`](delete.html) statements within a single [transaction](https://www.cockroachlabs.com/docs/dev/transactions.html). Executing statements in parallel helps reduce aggregate latency and improve performance. 
+
+<div id="toc"></div>
+
+## Why Use Parallel Statement Execution
+
+To understand why parallel statement execution is required, let's first understand sequential execution of SQL statements.
+
+Consider a scenario where we want to update a user's last name, favorite movie, and favorite song on a social networking application. Suppose the database has three tables: users, favorite_movies, and favorite_songs, that need to be updated to update the user information. Then the traditional transaction to update the user's information is as follows:
```

Comment:
@Amruta-Ranade, I think we need to be even more direct in this section and shouldn't lean so heavily on an example to explain "why". 

Perhaps we could repurpose/condense the content in this section to something like this:

> SQL engines traditionally execute the statements in a transaction sequentially: The server executes one statement to completion, sends the return value to the client, and only then starts the next statement, as demonstrated by the following conceptual diagram: 
>
> DIAGRAM
>
> In the case of a traditional single-node SQL database, execution of statements never leaves the single machine and so there isn't much communication latency. However, in the case of a multi-node replicated database like CockroachDB, execution of statements can span multiple machines, with communication and coordination between machines resulting in a higher cumulative latency. In such a case, executing SQL statement sequentially can add even more latency.
>
> THEN NEED TO END WITH WHY USE PARALLEL.